File: src/things/NPC.java
package things;

import java.util.ArrayList;

public class NPC extends Cell {

	private String chat = "";
	private ArrayList<Item> items = new ArrayList<Item>();

	public NPC(String name,String chat) {
		super(name);
		this.chat = chat;
	}


	public void Itemget(String name,int num){
		items.add(new Item(name));
	}

//	public void Itemgive(int index,int num){
//		items.get(index).getNumOf(num);
//	}

	public void Talk(){
		System.out.println(chat);
	}

}

File: src/cells/NPC.kt
package cells

import java.util.ArrayList

class NPC(name: String, chat: String) : Cell(name) {

    //	public void itemGive(int index,int num){
    //		items.get(index).getNumOf(num);
    //	}

    var chat = ""
    private val items = ArrayList<Item>()

    init {
        this.chat = chat
    }

    fun itemGet(name: String, num: Int) {
        items.add(Item(name, num))
    }
}
